backend/templates/model-na-31-1.typ

```diff
@@ -0,0 +1,309 @@
+  columns: (8em, 1fr, 1fr, 1fr, 8em),
+  headers: ("Voorletters", "Achternaam", "Rol", "Aanwezig (van - tot)", "Locatie"),
+  values: ("", "", "", "-", ""),
+  rows: 24,
+)
+
+#pagebreak(weak: true)
+
+== Getelde stembureaus
+
+=== De resultaten van onderstaande stembureaus zijn door het #location_type gecontroleerd en opgeteld tot het totaal van #is_municipality[de gemeente][het openbaar lichaam]. Als extra onderzoeken hebben plaatsgevonden wordt dat hier aangegeven. Als dat heeft geleid tot een correctie, en er dus een corrigendum is, wordt dat in de laatste kolom aangegeven.
+
+#light_table(
+  columns: (5em, 1fr, 1fr, 6em, 6em, 6em),
+  headers: (
+    [Nr.],
+    [Naam locatie],
+    [Postcode + Adres],
+    [Onderzocht vanwege een onverklaard verschil?],
+    [Onderzocht vanwege (een vermoeden van) een andere fout?],
+    [Uitslag gecorrigeerd?],
+  ),
+  values: input
+    .polling_stations
+    .map(polling_station => {
+      (
+        [#polling_station.number],
+        [#polling_station.name],
+        [
+          #if "polling_station_type" in polling_station and polling_station.polling_station_type == "Mobile" [
+            _(Mobiel stembureau)_
+          ] else [
+            #polling_station.address \
+            #polling_station.postal_code #polling_station.locality
+          ]
+        ],
+        align(center, checkbox(large: false)[]),
+        align(center, checkbox(large: false)[]),
+        align(center, checkbox(large: false)[]),
+      )
+    })
+    .flatten(),
+)
+
+=== Voor de stembureaus waar onderzoek naar is gedaan vanwege een vermoedelijke fout, maar waar geen correctie heeft plaatsgevonden, kan hieronder een toelichting worden gegeven. Het geven van een toelichting is niet verplicht.
+
+#empty_table(
+  columns: (7em, 1fr),
+  headers: ("Nummer stembureau", "Toelichting"),
+  values: ("", ""),
+  rows: 16,
+)
+
+#pagebreak(weak: true)
+
+== Tijdens de zitting
+
+=== Schrijf alle *bezwaren* van aanwezigen op.
+
+Bijvoorbeeld over het ongeldig verklaren van een stembiljet. Schrijf geen namen of andere persoonsgegevens op. Schrijf alle bezwaren op, ook als u het er niet mee eens bent. Geef aan hoe het bezwaar door het #location_type is behandeld.
+
+#empty_table(
+  columns: (7em, 1fr, 1fr),
+  headers: ("Tijdstip", "Bezwaar", [Reactie #location_type]),
+  values: ("", "", ""),
+  rows: 10,
+)
+
+=== Andere *bijzonderheden* die mogelijk invloed hebben op het telproces of de resultaten van het #location_type.
+
+Bijvoorbeeld een schorsing of als er meerdere verkiezingen tegelijk werden georganiseerd en een stembiljet in de verkeerde stembus zat.
+
+#empty_table(
+  columns: (7em, 1fr),
+  headers: ("Tijdstip", "Bijzonderheid"),
+  values: ("", ""),
+  rows: 5,
+)
+
+#pagebreak(weak: true)
+
+= Telresultaten van #is_municipality[de gemeente][het openbaar lichaam]
+
+== Aantal kiesgerechtigden
+
+#letterbox("Z", value: input.election.number_of_voters)[Kiesgerechtigden]
+
+== Toegelaten kiezers
+
+#is_municipality[
```

Comment:
It turns out that this conditional breaks when `input.summary.voters_counts.voter_card_count` is not present. It seems like Typst always tries to evaluate both branches of the `#is_municipality`.

I've fixed this for model Na 31-2 as follows:
https://github.com/kiesraad/abacus/pull/1795/files#diff-5a630b7b465c6c18bc40a85e337a5ef2d3d08e1fa67ade4196a9f43cfb85843b